<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_layout</class>
 <widget class="QWidget" name="main_layout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>818</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="top_button_layout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="toggle_project_info_btn">
       <property name="text">
        <string>∧</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="project_info_group">
     <property name="title">
      <string>项目信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="project_buttons_layout">
        <item>
         <widget class="QPushButton" name="new_project_btn">
          <property name="text">
           <string>新建项目</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_project_btn">
          <property name="text">
           <string>打开项目</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_project_btn">
          <property name="text">
           <string>修改项目</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_project_btn">
          <property name="text">
           <string>删除项目</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="row1_layout" stretch="0,3,0,1">
        <item>
         <widget class="QLabel" name="owner_label">
          <property name="text">
           <string>业主名称*：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="owner_input"/>
        </item>
        <item>
         <widget class="QLabel" name="project_number_label">
          <property name="text">
           <string>项目编号：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="project_number_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="row2_layout" stretch="0,3,0,1">
        <item>
         <widget class="QLabel" name="project_name_label">
          <property name="text">
           <string>项目名称*：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="project_name_input"/>
        </item>
        <item>
         <widget class="QLabel" name="department_label">
          <property name="text">
           <string>所属部门：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="department_input"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="row3_layout" stretch="0,0,0,0">
        <item>
         <widget class="QLabel" name="contractor_label">
          <property name="text">
           <string>工程总包方：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="contractor_input"/>
        </item>
        <item>
         <widget class="QLabel" name="create_date_label">
          <property name="text">
           <string>建立日期：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="date_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="row4_layout" stretch="0,1,0">
        <item>
         <widget class="QLabel" name="project_path_label">
          <property name="text">
           <string>项目路径*：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="project_path_input"/>
        </item>
        <item>
         <widget class="QPushButton" name="project_path_button">
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    font-size: 14px;
    padding: 0px 6px;
    border: 1px solid #ccc;
    background-color: #f0f0f0;
}
QPushButton:hover {
    background-color: #e0e0e0;
}
</string>
          </property>
          <property name="text">
           <string>…</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="confirm_project_layout">
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="confirm_project_btn">
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="product_info_group">
     <property name="title">
      <string>产品信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QTableWidget" name="product_table">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="rowCount">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>6</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>29</number>
            </attribute>
            <column>
             <property name="text">
              <string>序号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>设备名称*</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>设备位号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>产品编号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>设计阶段</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>设计版次</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="product_buttons_layout">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="modify_product_btn">
            <property name="text">
             <string>修改产品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_product_btn">
            <property name="text">
             <string>删除产品</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="confirm_product_btn">
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,2,4">
       <item>
        <widget class="QGroupBox" name="product_definition_group">
         <property name="title">
          <string>产品定义</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8">
            <item>
             <widget class="QLabel" name="product_type_label">
              <property name="text">
               <string>类型*：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="product_type_combo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,8">
            <item>
             <widget class="QLabel" name="product_form_label">
              <property name="text">
               <string>型式*：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="product_form_combo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,8">
            <item>
             <widget class="QLabel" name="product_model_label">
              <property name="text">
               <string>型号 ：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="product_model_input"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,8">
            <item>
             <widget class="QLabel" name="drawing_prefix_label">
              <property name="text">
               <string>图号 ：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="drawing_prefix_input"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="work_information_group">
         <property name="title">
          <string>工作信息</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="design_label">
              <property name="text">
               <string>设计    ：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="design_input"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="proofread_label">
              <property name="text">
               <string>校对    ：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="proofread_input"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="review_label">
              <property name="text">
               <string>审核    ：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="review_input"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="standardization_label">
              <property name="text">
               <string>标准化：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="standardization_input"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="approval_label">
              <property name="text">
               <string>批准    ：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="approval_input"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="co_signature_label">
              <property name="text">
               <string>会签    ：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="co_signature_input"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QFrame" name="image_area">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="image_label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="confirm_definition_layout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="confirm_definition_btn">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
